apps/web/src/styles/7-components/InfoBox/InfoBox.scss

```diff
@@ -0,0 +1,14 @@
+@use "govuk/settings/_colours-applied" as colors;
+
+$pi-govuk-background-colour:  colors.govuk-colour("light-grey");
+$pi-color: rgb(11, 12, 12);
+
+.pi-info-box {
+	background-color: $pi-govuk-background-colour;
+	border-top: solid colors.$govuk-border-colour 1px;
+	border-bottom: solid colors.$govuk-border-colour 1px;
+	color: $pi-color;
+	display: block;
+	padding: 10px;
```

Comment:
We have utility classes for this in the [DS](https://design-system.service.gov.uk/styles/spacing/). So remove it from here and use a class.

Reply:
Although I think you already did so not sure what this is for.

Reply:
you need to talk me through this ...

Reply:
In the NJK file of the component you already have `<div class="pi-info-box govuk-table govuk-!-padding-9">` padding added via `govuk-!-padding-9`.

Why do you need to add it again here?